bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -1393,7 +1393,11 @@ private Long getResultEntry(Map<String, @Nullable String> resultInput, String re
         }
 
         try {
-            result = Long.valueOf(resultInput.get(requestedKey));
+            String resultString = resultInput.get(requestedKey);
+            if (resultString == null) {
+                throw new NumberFormatException();
```

Comment:
Maybe add a message for the NumberFormatException here, the catch would end up using it.